### ISSUE TITLE
Develop ProgCode Organization Information Kit

### ISSUE THREAD

**Issue by jpb5013** [ProgressiveCoders/functions#172]
#### Description

Need to have a single source of truth that can be used to convey key ProgCode information, including mission, vision, plan, objective/strategy/goals, logo, tag-line, etc. Document should be used by multiple audiences, including press, community, partners. etc.
...

#### Problem

This information is currently scattered across a wide variety of content and mediums, making it very difficult to find information
...

#### Benefit

Having a single source that people use to find information about ProgCode will ensure people are able to understand key information, maintain a consistent focus, and be as impactful as possible
...

#### Plan

- [ ] Create [Organization Information Kit](https://docs.google.com/document/d/113YZ-N_p9edCJq99faw-xpapx19f3KDjLhWK1ADXqwg/edit?usp=sharing) document with outline
- [ ] Share with #operations
- [ ]  Add to [General Network Strategy Wiki](https://github.com/ProgressiveCoders/functions/wiki/General-Network-Strategy)

...

#### Decision Making

Vote by: Consensus
...

**Comment by julianrad** [comment]
I think we can get away with something a little simpler - in fact, I'd propose the following:

01 Our Mission. A clean and straightforward statement of the mission of Prog Code.
02 Our Values. A short paragraph describing how we go about achieving the mission.
03 Our Founders. Prog Code is people and the founders are where that culture begins.
04 EVERYTHING ELSE. Instead of a bone fide "media kit" I would fashion this more like social media assets. Logos, case studies, white papers, snackable video, photos from events, etc. If everything's online, we can more easily distribute and link and create connections for potential media, publicity, et al.

**Comment by rcscastillo** [comment]
## From Tino

Branding. Our name is going to be out there. We're not recognizable when our logo is put out there. 

Brand kit, along with the PR kit

**Comment by julianrad** [comment]
Agree. So far as future vision goes, I'd like to start asking sites/partners with whom we work to include our logo on their sites, kind of a "built by" or "built with" label that would link back to our website.

**Comment by jpb5013** [comment]
I think it's good to start with that, @julianrad, though in general we just need a single source of truth for all of this information. With time I'd expect it'd break into multiple different things.

And @tktino, I called this Information Kit to encompass the PR elements as well. As noted, when it gets larger we should consider making them separate elements.

**Comment by julianrad** [comment]
@jpb5013 any ideas on where that single source would come from? I'm curious about your vision of how this would work?

**Comment by jpb5013** [comment]
Just that document I linked to in the plan. It doesn't need to be anything grandiose.

**Comment by julianrad** [comment]
This one? https://docs.google.com/document/d/113YZ-N_p9edCJq99faw-xpapx19f3KDjLhWK1ADXqwg/edit?usp=sharing

**Comment by jpb5013** [comment]
Yes indeed. Haven't had a chance to put anything behind it yet, but ideally that'd be the starting place.
